conf/defaults.ini

```diff
@@ -640,6 +640,10 @@ managed_identity_enabled = false
 # Should be set for user-assigned identity and should be empty for system-assigned identity
 managed_identity_client_id =
 
+#################################### Role-based Access Control ###########
```

Comment:
Just to double check, when `custom.ini` is provided, does Grafana check first the custom file and fallbacks to defaults.ini for anything not defined in custom one?